### DISCUSSION TITLE
When SoftEtherVPN support ikev2?

### DISCUSSION THREAD

**Discussion by c998877** [SoftEtherVPN/SoftEtherVPN#1889] [Ideas]
When SoftEtherVPN support ikev2?

**Comment by chipitsine** [comment]
when someone will implement it. currently, no developer is working on it as far as I know

**Reply by c998877** [reply]
But the software version is still being updated. Last updated in June 2023

**Reply by metalefty** [reply]
>  no developer is working on it

"It" meant here is IKEv2. To be clear, let's replace "it" with IKEv2, "no developer is working on IKEv2". 

There's no relationship between IKEv2 support and other parts being updated. It won't happen until someone work on IKEv2. Why not become someone? I believe we're open.

**Reply by chipitsine** [reply]
we would be glad to accept IKEv2 contribution indeed

**Reply by Evengard** [reply]
Honestly, this is a dupe of https://github.com/SoftEtherVPN/SoftEtherVPN/issues/13 and all reasons about the current state of things were given there. Please read and contributions are welcome.
